include/layers/InputLayer.hpp

```diff
@@ -0,0 +1,169 @@
+#pragma once
+#include <algorithm>
+#include <cmath>
+
+#include "layers/Layer.hpp"
+
+namespace itlab_2023 {
+
+enum LayInOut {
+  kNchw,  // 0
+  kNhwc   // 1
+};
+
+class InputLayer : public Layer {
+ private:
+  LayInOut layin_;
+  LayInOut layout_;
+  int mean_;
+  int std_;
+
+ public:
+  InputLayer() = default;
+  InputLayer(LayInOut layin, LayInOut layout, int mean, int std) {
```

Comment:
```suggestion
  InputLayer(LayInOut layin, LayInOut layout, int mean = 0, int std = 1) {
```
In case that we won't need normalization